test/sql/copy/parquet/parquet_row_number.test

```diff
@@ -0,0 +1,16 @@
+# name: test/sql/copy/parquet/parquet_row_number.test
+# description: Test the filename option of the parquet reader
+# group: [parquet]
+
+require parquet
+
+query III
+select min(file_row_number), max(file_row_number), avg(file_row_number) from parquet_scan('data/parquet-testing/lineitem-top10000.gzip.parquet', file_row_number=1, filename=1);
```

Comment:
Could we test here with a filter by row number?

Reply:
added this, also added a test where the first couple of row groups are eliminated using parquet stats, then the file_row_number also needs to produce the correct result